opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/dto/result/body/FacadeToProtocolMapper.java

```diff
@@ -0,0 +1,10 @@
+package de.adorsys.opba.protocol.api.dto.result.body;
+
+import de.adorsys.opba.tppbankingapi.ais.model.generated.AccountList;
+import org.mapstruct.Mapper;
+
+@Mapper(implementationPackage = "de.adorsys.opba.protocol.api.dto.result.body.generated")
```

Comment:
It is better to extract:
`de.adorsys.opba.protocol.api.dto.result.body.generated` to constant

Reply:
done